### PR TITLE
Adding node-fetch target

### PR DESCRIPTION
**Pull request by hirenoble**
Hi Team,

As npm `request` package is no longer supported, it is essential to have code samples generated using `node-fetch`. I have attempted to write a target using `node-fetch`.

### CODE REVIEW

**Comment by darrenjennings** [comment]
thanks for the PR @hirenoble haven't had time to check it out but this is a welcome target!

**Comment by juliaqiuxy** [comment]
@darrenjennings Been quite sometime since this PR was created. Anything I can help with to get this merged? Thanks 🙏

**Comment by philsturgeon** [comment]
Anyone I can bribe with a tree donation to get this moving again?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (reynolek) [inline .vscode/settings.json:1]: unnecessary file

**Review by reynolek — APPROVED** [review]
Other than the accidental addition of `.vscode/settings.json`, looks good.

**Comment by jgiovaresco** [comment]
@darrenjennings @develohpanda it seems that the PR is almost done it just need to be rebased and we need to remove `.vscode/settings.json`
I'm wondering how I can help you on that ? Unfortunately I don't see the branche therefore I can't make modifications, do you have any idea ?

**Comment by develohpanda** [comment]
I'll have a look at this PR and merge it next week, then remove the settings file. 👍

**Comment by develohpanda** [comment]
Didn't notice the conflicts initially - I probably won't get to it anytime soon.

@jgiovaresco, if you are still keen to work on this, as there hasn't been any activity from the PR creator I'd say to grab the commit from his fork (https://github.com/hirenoble/httpsnippet), fix up whatever needs to be done, and create a standalone PR from your fork. We can attribute the feature to both once it gets merged in.

**Comment by develohpanda** [comment]
Closing in favor of #180. Thanks for laying the groundwork!

**Comment by hirenoble** [comment]
> Didn't notice the conflicts initially - I probably won't get to it anytime soon.
> 
> @jgiovaresco, if you are still keen to work on this, as there hasn't been any activity from the PR creator I'd say to grab the commit from his fork (https://github.com/hirenoble/httpsnippet), fix up whatever needs to be done, and create a standalone PR from your fork. We can attribute the feature to both once it gets merged in.

@develohpanda I don't see myself appearing in the contributors list, anyways I'm glad that this has been complete. 👍

**Comment by develohpanda** [comment]
> @develohpanda I don't see myself appearing in the contributors list, anyways I'm glad that this has been complete. 👍 

Will be in changelogs once we push out a new release that includes this feature, we don't have any control over what github lists as contributors though, which AFAIK is automatically produced from the authors of merged PRs 🤗

**Comment by hirenoble** [comment]
@develohpanda I opened this PR in November 2019, I waited for almost a year to get this pushed, we could have at least waited for a week for my response before rushing the things 😅

**Comment by develohpanda** [comment]
Apologies 🙏 Given there was no activity since the review in August, I had assumed you were MIA, and assumed you would have received notifications of the activity over the last few weeks as you were the PR author. In retrospect, you were not tagged in any conversations so may have missed it.

No harm intended, there was an offer for help and I took that offer with the information I had in hand. 

I believe if I manually change the author for the commit and force-push, it should attribute correctly, so I'll give that a go.

**Comment by hirenoble** [comment]
@develohpanda No worries at all, it was my first ever open source contribution so was pretty excited about that. If we can ensure this doesn't happen with anyone else, at least @mention would be super helpful. I get a ton of github notifications so may have missed, I need to get better at that too. 🥂

**Comment by develohpanda** [comment]
Duly noted 🌟 Please don't be discouraged to contribute to OSS because of this! I am exploring ways to ensure the correct author is attributed in cases like these where a PR was simply finalized by somebody else.

If you'd like to contribute some more, pop over to https://github.com/Kong/insomnia as we have some issues tagged with `Hacktoberfest`, which is running at the moment!

Also, I _think_ the GitHub UI ~is just catching up~ has just caught up! Your comments here have you tagged as `Contributor` as well as `Author`, I see you listed as a contributor on the repository. 👍

**Pull request closed without merging**